.buildkite/libbeat/pipeline.libbeat.yml

```
@@ -126,11 +128,12 @@ steps:
           automatic:
             - limit: 1
         agents:
-          provider: "gcp"
-          image: "${IMAGE_UBUNTU_X86_64}"
-          machineType: "${GCP_DEFAULT_MACHINE_TYPE}"
+          provider: "aws"
+          image: "${IMAGE_UBUNTU_X86_64_FIPS}"
+          instanceType: "m5.2xlarge"
```

If possible, let's use a smaller instance type (4 CPUs and 16 GB of RAM for UTs might be enough)
```suggestion
          provider: "aws"
          image: "${IMAGE_UBUNTU_X86_64_FIPS}"
          instanceType: "m5.xlarge"
```